pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -1125,7 +1126,7 @@ func (c *Cluster) persistExpectedMonDaemons() error {
 	}
 
 	configMap.Data = map[string]string{
-		EndpointDataKey: FlattenMonEndpoints(c.ClusterInfo.Monitors),
+		EndpointDataKey: endpoint.FlattenMonEndpoints(c.ClusterInfo.Monitors),
```

Comment:
not realted to the above issue

Reply:
Sorry can you expand on this? `FlattenMonEndpoints` now does not exist in the `mon` package, I moved it to a new `endpoint` package. I couldn't leave it in `mon` because of a circular dependency with the `controller` package.